src/main/java/com/amazon/aws/iot/greengrass/configuration/common/DeploymentCapability.java

```diff
@@ -6,5 +6,7 @@
 package com.amazon.aws.iot.greengrass.configuration.common;
 
 public enum DeploymentCapability {
-    LARGE_CONFIGURATION, LINUX_RESOURCE_LIMITS
+    LARGE_CONFIGURATION,
+    LINUX_RESOURCE_LIMITS,
+    SUBGROUP_DEPLOYMENTS
```

Comment:
this is a customer facing code/string right? is this the term we've settled on?

Reply:
Not yet. Expecting the terminology to be settled this week. Our UX team is driving that. We will have to revisit if this changes.